snapcraft/tests/repo/test_snaps.py

```diff
@@ -318,7 +319,15 @@ def test_refresh_fails(self):
         self.fake_snap_command.refresh_success = False
         self.assertRaises(errors.SnapRefreshError, snap_pkg.refresh)
 
+    def test_install_snaps_returns_revision(self):
+        installed_snaps = snaps.install_snaps(['fake-snap'])
+        self.assertThat(
+            installed_snaps,
+            Equals(['fake-snap=test-fake-snap-revision']))
+
     def test_install_multiple_snaps(self):
+        self.fake_snapd.server.RequestHandlerClass.installed_snaps = [
+            'new-fake-snap']
```

Comment:
Hmmm... the test is called `test_install_multiple_snaps`, but you seem to be adding `new-fake-snap` to the list of already installed snaps - so only one snap would be installed now?

It might just be the naming that's confusing here. I'm not sure what this is testing.

Reply:
This is just to make the test pass. Installed snaps is only queries the second time that the snaps endpoint is called. I don't have a better way to do it with the current fake.
I'm proposing a refactor soon, as a follow up branch.

Reply:
I see. A FIXME would've made that clearer. But I've seen the refactor now, so nevermind.